src/quota.rs

```diff
@@ -110,12 +110,13 @@ pub fn needs_quota_warning(curr_percentage: u64, warned_at_percentage: u64) -> b
 impl Context {
     // Adds a job to update `quota.recent`
     pub(crate) async fn schedule_quota_update(&self) -> Result<()> {
-        job::kill_action(self, Action::UpdateRecentQuota).await?;
-        job::add(
-            self,
-            job::Job::new(Action::UpdateRecentQuota, 0, Params::new(), 0),
-        )
-        .await?;
+        if !job::action_exists(self, Action::UpdateRecentQuota).await? {
+            job::add(
+                self,
+                job::Job::new(Action::UpdateRecentQuota, 0, Params::new(), 0),
+            )
+            .await?;
+        }
```

Comment:
```suggestion
        boolean exists_before = job::action_exists(self, Action::UpdateRecentQuota).await?;
        job::kill_action(self, Action::UpdateRecentQuota).await?;
        job::add(
            self,
            job::Job::new(Action::UpdateRecentQuota, 0, Params::new(), exists_before ? 10 : 0),
        )
        .await?;
```

Reply:
The problem is, with this solution, everytime `get_connectivity_html()` is called the job would be postponed to 10 seconds from "now".

So, the job fails once, get_connectivity_html() is called, the job is scheduled for in 10s, but for whatever reason get_connectivity_html() is called again before the 10s are over, the job is scheduled for in 10s again, and so on, until there is an interval of 10s where get_connectivity_html() is not called.